README.md

```diff
@@ -73,14 +73,9 @@ go test -failfast -race ./...
 ```
 
 #### C++
-The `gtest` test suite is an extension to main developer test suite. `C++` code is
-a deeper internal code and is more prone to change than `go` one, so at the
-moment it can even be considered private for the project. Some features however
-are inconvenient to test directly from `go`, but are important enough to be
-tested on their own.
-
-If functionality can be tested reasonably well through public `go` interface,
-use it in order to limit the number of "private" tests.
+The `gtest` test suite is a test suite for `C++` code. When choosing which test
```

Comment:
When choosing where to implement new tests...? Now is sounds like you have a choice in which suite to run, as if it doesn't matter if you run go tests or cpp tests